griptape/tasks/code_execution_task.py

```diff
@@ -1,19 +1,18 @@
 from __future__ import annotations
 
-from typing import Callable
+from typing import TYPE_CHECKING, Callable
 
 from attrs import define, field
 
-from griptape.artifacts import BaseArtifact, ErrorArtifact
 from griptape.tasks import BaseTextInputTask
 
+if TYPE_CHECKING:
+    from griptape.artifacts import BaseArtifact
+
 
 @define
 class CodeExecutionTask(BaseTextInputTask):
     run_fn: Callable[[CodeExecutionTask], BaseArtifact] = field(kw_only=True)
 
     def run(self) -> BaseArtifact:
-        try:
-            return self.run_fn(self)
-        except Exception as e:
-            return ErrorArtifact(f"error during Code Execution Task: {e}")
+        return self.run_fn(self)
```

Comment:
other tasks dont do this do they?

Reply:
They do not which is why I removed it. `BaseTask` [will catch this already](https://github.com/griptape-ai/griptape/blob/f2b529ab361c33b2a26a4510f78f85181b96af4f/griptape/tasks/base_task.py#L163-L178).